### PR TITLE
Panzer: Make disc-fe Epetra optional

### PR DESCRIPTION
**Pull request by JacobDomagala**
There're lots of whitespace changes in this PR (removed trailing whitespaces), for easier reviewing I'd suggest turning whitespace changes off in GitHub.
Screenshot from 2022-05-13 14-56-02

### CODE REVIEW

**Review by rppawlo — APPROVED** [review]
Looks good to me! Thanks @JacobDomagala !

File: packages/panzer/disc-fe/src/evaluators/Panzer_ScatterDirichletResidual_Epetra.cpp [hunk 1]
```diff
@@ -40,10 +40,11 @@
 // ***********************************************************************
 // @HEADER
 
-#include "Kokkos_View_Fad.hpp"
-
 #include "PanzerDiscFE_config.hpp"
 
+#ifdef PANZER_HAVE_EPETRA
+
+#include "Kokkos_View_Fad.hpp"
```
[the remark this answers is not in the document]
Comment (JacobDomagala) [reply packages/panzer/disc-fe/src/evaluators/Panzer_ScatterDirichletResidual_Epetra.cpp:47]: Yes that's a good idea. All headers/sourced are globbed, but doing list(remove ...) should do the trick.

**Comment by JacobDomagala** [comment]
@rppawlo  Ok this should be ready to review again

**Comment by JacobDomagala** [comment]
@rppawlo Ok, let's run the CI testers

**Pull request merged**